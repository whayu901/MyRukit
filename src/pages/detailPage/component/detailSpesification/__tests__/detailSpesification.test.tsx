import "react-native";
import React from "react";
import renderer from "react-test-renderer";

import DetailSpesification from "../index";

jest.useFakeTimers();

// Note: test renderer must be required after react-native.

describe("test detail spesification", () => {
  describe("rendering", () => {
    it("should match to snapshot", () => {
      const detailSpesification = renderer.create(
        <>
          <DetailSpesification
            species="test1"
            gender="laki-laki"
            status="alive"
          />
        </>,
      );
      expect({
        detailSpesification,
      }).toMatchSnapshot("Text DetailSpesification snapshot true");
    });
  });
});
