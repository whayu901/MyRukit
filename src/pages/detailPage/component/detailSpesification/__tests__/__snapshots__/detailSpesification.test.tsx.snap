// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test detail spesification rendering should match to snapshot: Text DetailSpesification snapshot true 1`] = `
Object {
  "detailSpesification": <View
    style={
      Object {
        "backgroundColor": "#EF5DA8",
        "borderRadius": 10,
        "padding": 10,
      }
    }
  >
    <View
      style={
        Object {
          "flexDirection": "row",
          "marginVertical": 5,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#ffffff",
            "fontFamily": "Lekton-Bold",
            "fontSize": 14,
          }
        }
      >
        spesies:
      </Text>
      <Text
        style={
          Object {
            "color": "#ffffff",
            "fontFamily": "Lekton-Regular",
            "fontSize": 14,
            "marginLeft": 10,
          }
        }
      >
        test1
      </Text>
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
          "marginVertical": 5,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#ffffff",
            "fontFamily": "Lekton-Bold",
            "fontSize": 14,
          }
        }
      >
        gender:
      </Text>
      <Text
        style={
          Object {
            "color": "#ffffff",
            "fontFamily": "Lekton-Regular",
            "fontSize": 14,
            "marginLeft": 10,
          }
        }
      >
        laki-laki
      </Text>
    </View>
    <View
      style={
        Object {
          "flexDirection": "row",
          "marginVertical": 5,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#ffffff",
            "fontFamily": "Lekton-Bold",
            "fontSize": 14,
          }
        }
      >
        status:
      </Text>
      <Text
        style={
          Object {
            "color": "#ffffff",
            "fontFamily": "Lekton-Regular",
            "fontSize": 14,
            "marginLeft": 10,
          }
        }
      >
        alive
      </Text>
    </View>
  </View>,
}
`;
